test/services/projectList.spec.js

```diff
@@ -136,7 +142,9 @@ describe('services', () => {
       it('should not throw an error if the ID and path are specified', () => {
         const service = new ProjectListService();
         // eslint-disable-next-line prettier/prettier
-        expect(() => service.validateProjectListEntry({ id: '1', path: '/Test/Path' })).not.toThrow(Error);
+        expect(() => service.validateProjectListEntry({ id: '1', path: '/Test/Path' })).not.toThrow(
```

Comment:
I'm okay to accept the linter suggestion here, but we should remove the eslint-disable line above